website/pages/en/arbitrum/arbitrum-faq.mdx

```diff
@@ -20,12 +20,19 @@ The Graph community decided to move forward with Arbitrum last year after the ou
 
 ## What do I need to do to use The Graph on L2?
 
-Users bridge their GRT and ETH  using one of the following methods:
+The Graph’s billing system accepts GRT on Arbitrum, and users will need ETH on Arbitrum to pay their gas. While The Graph protocol started on Ethereum Mainnet, all activity, including the billing contracts, is now on Arbitrum One.
 
-- [The Arbitrum Bridge](https://bridge.arbitrum.io/?l2ChainId=42161)
-- [TransferTo](https://transferto.xyz/swap)
-- [Connext Bridge](https://bridge.connext.network/)
-- [Hop Exchange](https://app.hop.exchange/#/send?token=ETH)
+Consequently, to pay for queries, you need GRT on Arbitrum. Here are a few different ways to achieve this:
+
+- If you already have GRT on Ethereum, you can bridge it to Arbitrum. You can do this via the GRT bridging option provided in Subgraph Studio or by using one of the following bridges:
+  - [The Arbitrum Bridge](https://bridge.arbitrum.io/?l2ChainId=42161)
+  - [TransferTo](https://transferto.xyz/swap)
+  - [Connext Bridge](https://bridge.connext.network/)
+  - [Hop Exchange](https://app.hop.exchange/#/send?token=ETH)
+- If you have other assets on Arbitrum, you can swap them for GRT through a swapping protocol like Uniswap.
+- Alternatively, you can acquire GRT directly on Arbitrum through a decentralized exchange.
+
+Once you have GRT on Arbitrum, you can add it to your billing balance.
```

Comment:
Should this be a link to the section in the `/billing` page, instead of duplicating the information?